test/e2e/utils/utils.go

```diff
@@ -79,14 +79,24 @@ func WaitForPodFailure(f *framework.Framework, name string, timeout time.Duratio
 			case v1.PodFailed:
 				return true, nil
 			case v1.PodSucceeded:
-				return true, fmt.Errorf("pod %q successed with reason: %q, message: %q", name, pod.Status.Reason, pod.Status.Message)
+				return true, errPodSucceeded{name, pod.Status.Reason, pod.Status.Message}
```

Comment:
It's enough to use `errors.Errorf()` instead of `fmt.Errorf()` here.

Reply:
Oops. Idk why goerr113 I put was missing in the commit.! Let me put it.!